tests/xtimer_mutex_lock_timeout/main.c

```diff
@@ -42,9 +49,58 @@ static const shell_command_t shell_commands[] = {
       cmd_test_xtimer_mutex_lock_timeout_long_unlocked, },
     { "mutex_timeout_long_locked", "locked mutex with long timeout",
       cmd_test_xtimer_mutex_lock_timeout_long_locked, },
+    { "mutex_timeout_long_locked_low",
+      "locked mutex from lower prio thread function with long timeout",
```

Comment:
actually the description part of the command is confusing as its not really explaining what happens here. Somewhere in the test it should be explained in clear to understand language what the test checks, what kind of guarantee this implies and by that what it means if the test fails. Maybe for the short description in the command something like "checks if xtimer_mutex_lock_timeout can acquire a mutex that was previously locked by a lower prio thread" would be better to understand for a random dude looking at this test for the first time. Also the terms low and long are not very precise here. I guess you want to express something like *longer than needed* i.e. the timeout is big enough that we are absolutely sure the timeout should never actually be triggered if everything is working fine. And *too short* as in "the timeout is so small that we know it should always trigger if everything works.

Reply:
The description should be something short because a long one is difficult to read. If a "random dude" wants to know what the test does he should read the documentation. I would almost say the command and description is currently too long. I agree that the description is currently not perfect. What do you think about` locked mutex with long timeout tested with 2 threads`
The part: `locked mutex with long timeout` is to be consistent with the other commands.

**LONG** means the timer does not spin(timeout longer than `XTIMER_BACKOFF`).
NOT that is does not trigger. 
It should trigger in `cmd_test_xtimer_mutex_lock_timeout_long_locked`.
It is only called **LONG** because there will be a test with a short one (meaning the timer will spin). This test will be added as soon as the bugs are removed because it fails at the moment.

**tracking pr to show what my goal is (how I will remove the bugs)#11660**

**LOW** and **LONG** are not meant to be precise. It is only short for lower than main thread and longer than `XTIMER_BACKOFF`. How much longer/lower is irrelevant for the test.

To sum it up for me a help description should be short. I am not against changing the  description. And long and low are chosen on purpose.

Reply:
Ok thanks for the clarification. After understanding your intentions for the naming (long) I still think its not precise. I.e. as a reader you don't get the relevant information - that is "no-spin", how about using that instead?. 
>To sum it up for me a help description should be short.

I agree. but it should also be as clear as possible because otherwise its no help ;) How about `lock low-prio-locked-mutex from high-prio-thread (no-spin timeout)`

Reply:
(the dashes are meant to make it more clear which word modifies what)

Reply:
`mutex_timeout_long_locked_low lock low-prio-locked-mutex from high-prio-thread (no-spin timeout)` is a little difficult to read

Reply:
I changed it.

Reply:
True - but if you know where the description starts its not really worse than before. and at least it more unambiguous...

Reply:
Would you also change the "long" part of the other descriptions to "no-spin" to make it consistent again?

Reply:
Yes.